index.js

```diff
@@ -15,16 +29,16 @@ const aliased = [
   'utils',
 ];
 
-function isInternalModule({ moduleName }) {
-  return moduleName.startsWith('@qsi/');
+function isScopedModule({ moduleName }) {
```

Comment:
I've got a few PR's open on the underlying library so i'm hoping that will give us more options in the future as well.